### DISCUSSION TITLE
Nats Service Authentication and Agents never Update

### DISCUSSION THREAD

**Discussion by mr6volt** [amidaware/tacticalrmm#1341] [General]
This is a new install using Ubuntu 20.04

Any agents installed on endpoints will show up on the dashboard, and Mesh functionality works fine.

RMM Functions fail, and the agent never updates the RMM with System info. (53 Years)

I discovered that the NATS Service is experiencing an authentication error:

`[11723] 2022/10/31 15:41:58.759026 [ERR] x.x.x.x/127.0.0.1:41066 - wid:21 - "v1.18.0:go:TacticalRMM" - authentication error - User "RANDOMLETTERS"`

I have encountered this issue twice in a row. (Blasted away the first Server Instance, and Attempted again)
Is this fixable?

**Comment by dinger1986** [comment]
Where have you installed this?

What's in your hosts file

**Reply by mr6volt** [reply]
Baremetal

Hosts file was never touched.

**Reply by dinger1986** [reply]
Ok `cat /etc/hosts` please

Feel free to submit to help@amidaware.com as it should have your domains in it
